src/main/java/org/example/models/Ticket.java

```diff
@@ -15,43 +15,85 @@ public class Ticket {
     private int id;
 
     @ManyToOne
-    @JoinColumn(name = "user_id", nullable = false)
+    @JoinColumn(name = "user_id", nullable = true) // Updated to nullable as the JSON has null for some user associations
     private User user;
 
-    @Enumerated(EnumType.ORDINAL)
+    @Enumerated(EnumType.STRING) // Changed to STRING for better readability in the database
     @Column(name = "ticket_type", nullable = false)
     private TicketType ticketType;
 
+    @Column(name = "ticket_class", nullable = true)
+    private String ticketClass; // Matches "ticketClass" in JSON
+
+    @Column(name = "start_date", nullable = true)
+    private LocalDate startDate; // Matches "startDate" in JSON
+
+    @Column(name = "price", nullable = true, precision = 10, scale = 2)
+    private BigDecimal price; // Matches "price" in JSON, allows null and decimals
+
     @CreationTimestamp
     @Column(name = "creation_date", nullable = false, updatable = false)
     private LocalDateTime creationDate;
 
     public Ticket() {
     }
 
-    public Ticket(User user, TicketType ticketType) {
+    public Ticket(User user, TicketType ticketType, String ticketClass, LocalDate startDate, BigDecimal price) {
         this.user = user;
         this.ticketType = ticketType;
+        this.ticketClass = ticketClass;
+        this.startDate = startDate;
+        this.price = price;
     }
 
+    // Getters and Setters
+
     public int getId() {
         return id;
     }
 
     public int getUserId() {
```

Comment:
Why do we need this method?)

Reply:
it's my random mistake of duplicationg the code)